lib/googlesearch.py

```diff
@@ -23,6 +25,7 @@ def closeBrowser():
         browser.quit()
 
 def search(req, stop):
+    time.sleep(5)
```

Comment:
What is this ?

Reply:
Adding this sleep command here helped me avoid being blocked by google. I've previously done some bot scripts and a small wait usually prevents our scripts from being detected by the website.